server/backend/mikro-orm.config.ts

```diff
@@ -0,0 +1,11 @@
+import { Options } from '@mikro-orm/core';
+
+const options: Options = {
+  entities: ['./src/**/*.entity.ts', './src/**/*.entity.js'],
+  type: 'postgresql',
+  dbName: process.env.DB_NAME || undefined,
+  host: process.env.DB_HOST || undefined,
```

Comment:
discussed offline: update README with cool stuff regarding these thigns

Reply:
Will update the README later when I set up Docker stuff, running locally isn't entirely necessary atm because we have integration tests